src/Routing/Router.php

```diff
@@ -1032,6 +1033,20 @@ public static function parseRoutePath(string $url): array
         $defaults['controller'] = $matches['controller'];
         $defaults['action'] = $matches['action'];
 
+        if ($matches['params'] !== '') {
+            $paramsArray = array_values(array_filter(explode('/', $matches['params'])));
```

Comment:
Is the `array_values()` necessary? Could the `array_filter()` be avoided if we `rtim()` the trailing slash instead?

Reply:
indeed